### ISSUE TITLE
Update to u20?

### ISSUE THREAD

**Issue by BigHummus69** [vidio-boy/Eaglercraft1.8.8#2]
Hey, did you update the files to eaglercraft u20?

**Comment by vidio-boy** [comment]
No, but I will soon, I'm working on updates right now :D

**Comment by antgame11** [comment]
updating a html file isnt hard LMFAO

**Comment by Ngamanthebest** [comment]
um im trying to boot up my world and its saying connection refused how do i fix it

**Comment by DanielV69** [comment]
@ant-7802 yes it IS!!!!!! you have to open eaglercraft then click update then upload that html to github!!!!!! that's so much workkkkk

(joke obv)

**Comment by BigHummus69** [comment]
Bro ain't no way this gotta at least be u40 bro
